src/oss-find-squats-lib/Mutators/IMutator.cs

```diff
@@ -37,14 +38,14 @@ public IEnumerable<Mutation> Generate(PackageURL arg)
                 if (hasNamespace)
                 {
                     yield return new Mutation(
-                        mutated: arg.CreateWithNewNames(arg.Name, mutation.Mutated).ToString(),
+                        mutated: arg.CreateWithNewNames(arg.Name, HttpUtility.UrlEncode(mutation.Mutated)).ToString(),
```

Comment:
Please add a test to protect against regressions here.

Reply:
I believe this should be addressed now.